src/runtime_src/xdp/profile/device/hal_device/xdp_hal_device.cpp

```diff
@@ -155,12 +155,12 @@ uint64_t HalDevice::getDeviceAddr(size_t id)
 
 double HalDevice::getMaxBwRead()
 {
-  return xclGetReadMaxBandwidthMBps(mHalDevice);
+  return 9600.0;
```

Comment:
Is this change (and the corresponding write change) needed? Would be good to avoid hard-coding the max BW values.

Reply:
This change is needed as these functions aren't defined in edge HAL and cause failure on dlopen. Currently these values are hardcoded everywhere. We need to clean that up in future.